src/components/Filters.js

```diff
@@ -0,0 +1,64 @@
+
+const Filters = props => {
```

Comment:
great idea to extract Filters logic outside of the component and all the logic is concise and readable